koku/api/report/queries.py

```diff
@@ -666,14 +676,13 @@ def _get_group_by(self):
         return group_by
 
     def _get_tag_group_by(self):
-        """Create list of tag based group by parameters."""
+        """Create list of tag-based group by parameters."""
         group_by = []
         tag_groups = self.get_tag_group_by_keys()
         for tag in tag_groups:
-            tag_db_name = self._mapper.tag_column + "__" + strip_prefix(tag, TAG_PREFIX)
-            tag = str.encode(tag)
-            tag = quote_from_bytes(tag, safe=URL_ENCODED_SAFE)
-            group_pos = self.parameters.url_data.index(tag)
+            sanitized_tag = sanitize_tag(tag)
```

Comment:
I think you need to strip the prefix.
```suggestion
            sanitized_tag = sanitize_tag(strip_prefix(tag, TAG_PREFIX))
```